### ISSUE TITLE
Segmentation fault on Linux since #6979

### ISSUE THREAD

**Issue by skielred** [ppy/osu#7024]
**Describe the crash:** 
The game window does not even open, the logs stop after the "Loading fonts xxxx" outputs (LogLevel is on Debug).
Not sure if it is related to #6979 or to the osu-framework update that comes with it, I tried a dozens of other commits before 3d58f1d920bf2b1769afb17e97dad74f8a2af531 (when #6979 was introduced) and the crash did not occur, I'm able to open the game properly.
But on 3d58f1d920bf2b1769afb17e97dad74f8a2af531 and every single commit above up to 4dc8e0ae20c8d84033bfbe2497ebf3e0315aca55, the crash occurs.

Am I missing something that can make the game outputs more logs? I'm running the game with `dotnet run --project osu.Desktop -v diag`
I'd like to debug this issue but I'm not sure where to start, I did my best narrowing on which update it started occuring, but I don't know what causes it.

**osu!lazer version:** 
Each one from 3d58f1d920bf2b1769afb17e97dad74f8a2af531 to 4dc8e0ae20c8d84033bfbe2497ebf3e0315aca55

**Computer Specifications:** 
Linux, 64-bits, Fedora, i3 on X11.

**Comment by bdach** [comment]
You're most likely hitting #7018, which has already been fixed framework-side at ppy/osu-framework#3057. Please confirm by checking `dmesg` output - if it's a segfault with `libbass_fx.so`, this is 99% the case.

**Comment by skielred** [comment]
```
segfault at 8 ip 00007f16843095d9 sp 00007f16b52d41e0 error 4 in libbass_fx.so[7f16842f8000+16000]
```
That's it! Thanks for the heads up! Glad it's fixed, closing this issue.
